integrations/cypress/cypress/support/commands/logout.ts

```diff
@@ -0,0 +1,51 @@
+import { Slice } from '@effection/atom';
+import { GetClientFromSpec, TestState } from '../types';
+
+export interface MakeLogoutOptions {
+  atom: Slice<TestState>;
+  getClientFromSpec: GetClientFromSpec;
+}
+
+export const makeLogout = ({ atom, getClientFromSpec }: MakeLogoutOptions) => () => {
+  return new Cypress.Promise((resolve, reject) => {
+    let client = getClientFromSpec(Cypress.spec.name);
+
+    Cypress.log({
+      name: 'simulacrum-logout',
+      displayName: 'simulacrum-logout',
+      message: 'in logout'
+    });
+
+    let simulation = atom.slice(Cypress.spec.name, 'simulation').get();
+
+    if(!client || !simulation) {
+      Cypress.log({
+        name: 'simulacrum-logout',
+        displayName: 'simulacrum-logout',
+        message: 'no simulation or client'
+      });
+      resolve();
+      return;
+    }
+
+    client.destroySimulation(simulation).then(() => {
+      Cypress.log({
+        name: 'simulacrum-logout',
+        displayName: 'simulacrum-logout',
+        message: 'simulation destroyed'
+      });
+
+      atom.slice(Cypress.spec.name).remove();
```

Comment:
Is this synchronous? or do we need to wait for this operation to finish?

Reply:
that is synchronous

Reply:
Got it.